erpnext_com/www/docs/user/manual/en/human-resources/shift-management.md

```diff
@@ -9,19 +9,80 @@ To use Shift Management in ERPNext,
+1. Human Resources > Shift Management > Shift Type
+2. Enter Name, Start Time and End Time
+3. Save
+<img class="screenshot" alt="Shift Type" src="{{docs_base_url}}/assets/img/human-resources/new-shift-type.png">
 
-* Enter Shift Type, Start Time and End Time for quick entry.
+## 2. Features
 
+In addition to defining the different shifts in your organization, the Shift Type document also has the setting for auto attendance. Auto attendance marks the attendance for the employees assigned to this shift based on records in the 'Employee Checkin' Document. Auto Attendance for all shift type records are attempted to be marked every hour. You can also trigger the auto attendance manually for a single shift type by pressing the 'Mark Auto Attendance' button in the shift type document.
 <img class="screenshot" alt="Shift Type" src="{{docs_base_url}}/assets/img/human-resources/shift-type.png">
 
-### Shift Request
+### 2.1 Start Time
+The time of the day when this shift starts. The time is to be entered in 24Hrs format.
+
+### 2.2 End Time
+The time of the day when this shift ends. The time is to be entered in 24Hrs format. 
+
+> Note: For cases where the 'End Time' is less that 'Start Time', the shift is assumed to be a night shift that starts on one calendar date and end on the next calendar date .
+
+### 2.3 Holiday List
+The Applicable Holidays for this shift can be selected here. If left blank then the default holiday list from the employee or the company document is taken in to account.
+
+### 2.4 Enable Auto Attendance
+You can use this option to enable marking of attendance for the employees assigned to this shift based on their 'Employee Checkin' records.
+
+### 2.5 Auto Attendance Settings
+Below are the settings you can use to configure the Auto Attendance as per your requirements.
+
+#### 2.5.1 Determine Check-in and Check-out
+Employee Check-in may not always have an IN/OUT log type. For, such scenarios you could use this option to get appropriate results from the auto attendance system.
+
+1. Alternating entries as IN and OUT during the same shift
+	- The first entry is taken as IN followed by the next entry as OUT and the following entry as IN and so on.
+2. Strictly based on Log Type in Employee Checkin
+	- The check-in is determined as IN or OUT strictly based on the 'Log Type' in the Employee Checkin record
+
+#### 2.5.2 Working Hours Calculation Based On
+Working hours can be calculated either by including the breaks in between the shift or by excluding the breaks. 
+
+This can be configured using the following option:
+
+1. First Check-in and Last Check-out
+	- Selecting this option calculates the working hour by considering the first IN and last OUT Employee Checkin during the shift.
+	- In case the IN/OUT is determined by Alternating entries then the first Employee Checkin is considered as IN and the last Employee Checkin is considered as OUT for the purpose of working hours calculation.
```

Comment:
```suggestion
	- In case the IN/OUT is determined by alternating entries then the first Employee Checkin is considered as IN and the last Employee Checkin is considered as OUT for the purpose of working hours calculation.
```